main.py

```diff
@@ -209,16 +213,28 @@ def publish_incidents():
 #     load_from_csv("loadtata_result.csv")
 #     return "success"
 
+import queue
+
+token_q = queue.Queue()
+
 
 @app.route("/token", methods=["PUT"])
-@limiter.limit("100/day;5/hour;1/minute")
 def register_token():
     deviceId = request.get_json().get("deviceId", None)
     token = request.get_json().get("token", None)
+    if not deviceId:
+        raise ValueError("No deviceId detected")
     if not token:
-        raise ValueError("no token detected ")
-    res = registerNewToken(deviceId, token)
+        raise ValueError("No token detected")
+
+    token_q.put(deviceId)
+    while not token_q.empty() and token_q.queue[0] != deviceId:
```

Comment:
what does this loop do? could it cause dead loop?

Reply:
they are unused old code, i will remove that